products/sle/main.pm

```diff
@@ -70,6 +70,10 @@ sub is_sles4sap () {
     return get_var('FLAVOR', '') =~ /SAP/;
 }
 
+sub is_smt () {
+    return (get_var("PATTERNS", '') || get_var('HDD_1', '')) =~ /smt/;
```

Comment:
@coolo updated, condition without brackets did work, but it was wrong ... any PATTERN would match